### ISSUE TITLE
Scaffold out Freestyle::Usage with support for :example named block

### ISSUE THREAD

**Issue by chrislopresto** [chrislopresto/ember-freestyle#377]
Make something like this:

```hbs
<Freestyle::Usage @title='More List Icon'>
  <:example>
    <MoreList
        @morePages={{this.morePages}}
        @drillDown={{fn @logAction 'MoreList drillDown'}}
    />
  </:example>
</Freestyle::Usage>
```

**Comment by lukemelia** [comment]
Scaffolded in #380

**Comment by lukemelia** [comment]
Implemented in #457
